index.js

```diff
@@ -4,22 +4,32 @@
  * @param {Object} store - An object representing a single store. See the instructions for details on its shape.
  * @returns {Object[]} The same `stores` array that was inputted.
  */
-function addNewStore(stores, store) {}
+function addNewStore(stores, store) {
+  stores.push(store)
+  return stores
+}
 
 /**
  * Removes a store object at the given position.
  * @param {Object[]]} stores - An array of store objects.
  * @param {number} index - A number representing the index of the store to be removed from the array.
  * @returns {Object[]} The same `stores` array that was inputted.
  */
-function removeStoreAtPosition(stores, index) {}
+function removeStoreAtPosition(stores, index) {
+  stores.pop(index)
+  return stores
+}
 
 /**
  * Creates a duplicate of the `store` object. No references should be shared between the inputted `store` and the result.
  * @param {Object} store - An object representing a single store. See the instructions for details on its shape.
  * @returns {Object} The duplicated store object. This should not be the same as the store that was inputted.
  */
-function duplicateStore(store) {}
+function duplicateStore(store) {
+  let newobj = {}
+  Object.assign(newobj,store)
+  return newobj
+}
```

Comment:
since you have objects and arrays as properties in the original store object. Those reference types also need to be created